pkg/northbound/gnmi/subscribe_test.go

```diff
@@ -239,10 +229,10 @@ func Test_WrongDevice(t *testing.T) {
 	targets["Device1"] = struct{}{}
 	//subs = append(subs, utils.MatchWildcardRegexp("/cont1a/*/leaf3c"))
 	//go listenForUpdates(changeChan, serverFake, mgr, targets, subs, resChan)
-	config1Value05, _ := devicechangetypes.NewChangeValue("/cont1a/cont2a/leaf2c", devicechangetypes.NewTypedValueString("def"), false)
-	config1Value09, _ := devicechangetypes.NewChangeValue("/cont1a/list2a[name=txout2]", devicechangetypes.NewTypedValueEmpty(), true)
-	change1, _ := change.NewChange([]*devicechangetypes.ChangeValue{config1Value05, config1Value09}, "Remove txout 2")
-	changeChan <- events.NewConfigEvent("Device1", change1.ID, true)
+	//config1Value05, _ := devicechangetypes.NewChangeValue("/cont1a/cont2a/leaf2c", devicechangetypes.NewTypedValueString("def"), false)
```

Comment:
don't know about commenting them out

Reply:
I'm leaving them there for reference for when the test is ported to have same behaviour